shard.yml

```
@@ -1,6 +1,8 @@
 name: immutable
 version: 0.1.24
 
+crystal: ">= 0.35, < 2.0"
```

I think keeping the version at `>= 0.35.0` for now is best until we know it fails for `>= 2`
https://forum.crystal-lang.org/t/artificial-shard-restrictions-version-2-0-0-considered-harmful/

```suggestion
crystal: ">= 0.35"
 ```

---

sounds good to me.